runner/src/exercism/test_runner.gleam

```diff
@@ -0,0 +1,113 @@
+// TODO: Get and show stacktrace
+import gleam/io
+import gleam/list
+import gleam/bool
+import gleam/string
+import gleam/option.{None, Some}
+import gleam/dynamic.{Dynamic}
+import gleam/erlang
+import gleam/erlang/atom.{Atom}
+import gleam/erlang/charlist.{Charlist}
+import simplifile
+import glance
+import exercism_test_runner/internal.{Error, Suite, Test, TestResult}
+
+pub fn main() {
+  let assert Ok(files) = read_directory("test")
+  let suites = list.map(files, read_module)
+  let results = list.flat_map(suites, run_suite)
+  let #(passed, message) = internal.print_summary(results)
+  io.println("\n" <> message)
+
+  let assert Ok(_) = case erlang.start_arguments() {
+    ["--json-output-path=" <> path] | ["--json-output-path", path] -> {
+      let json = internal.results_to_json(results)
+      simplifile.write(json, path)
+    }
+    _ -> Ok(Nil)
+  }
+
+  halt(case passed {
```

Comment:
Is this correct? From the test runner specification:
> The runner must exit with an exit code of 0 if it has run successfully, regardless of the status of the tests.

Reply:
Aye, there's a script that runs it and handles things like the exit status and compile errors

https://github.com/exercism/gleam-test-runner/blob/main/bin/run.sh